src/Estimators/tests/RobotDynamicsEstimator/SubModelKinDynWrapperTest.cpp

```diff
@@ -264,11 +264,11 @@ TEST_CASE("SubModelKinDynWrapper")
 
                     Eigen::MatrixXd jac = jacobian.block(0, 6, 6, subModelList[idx].getModel().getNrOfDOFs());
 
-                    for (int ftIndex = 0; ftIndex < subModelList[idx].getNrOfFTSensor(); ftIndex++)
+                    for (auto & [ftName, ftObj] : subModelList[idx].getFTList())
                     {
-                        if (subModelList[idx].getFTSensor(ftIndex).name == key)
+                        if (ftName == key)
                         {
-                            contactTorques.noalias() += (int)subModelList[idx].getFTSensor(ftIndex).forceDirection * jac.transpose() * val;
+                            contactTorques.noalias() += (int)ftObj.forceDirection * jac.transpose() * val;
```

Comment:
Consider using a ``static_cast``  (i.e. ``static_cast<int>(ftObj.forceDirection)``) instead of a classical casting. It is a little bit safer